src/frontend/styles/components/sidebar.pcss

```diff
@@ -73,11 +73,36 @@
     }
   }
 
+  &__search {
+    @apply --input;
+
+    appearance: none;
+    background: url("../svg/search.svg") left 10px center no-repeat;
+    background-color: var(--color-input-primary);
+    border: 1px solid transparent;
+    border-radius: 8px;
+    padding: 8.5px 35px;
+    padding-right: 42px;
+    line-height: 17px;
+
+    &-wrapper::after {
+      color: var(--color-button-secondary);
+      font-weight: 400;
+      font-size: 14px;
+      line-height: 17px;
+      content: attr(data-shortcut);
+      margin-left: -42px;
+    }
+  }
```

Comment:
lets hide the search on mobile devices since it is not useful there

<img width="392" alt="image" src="https://user-images.githubusercontent.com/3684889/195414445-534db7c8-9dc4-484d-ba43-14af2922d6f4.png">

Reply:
done